manage_proj_cat_edit_page.php

```diff
@@ -68,7 +68,7 @@
 print_manage_menu( 'manage_proj_cat_edit_page.php' );
 ?>
 
-<div class="col-md-12 col-xs-12" xmlns="http://www.w3.org/1999/html">
+<div class="col-md-12 col-xs-12" xmlns="http://www.w3.org/1999/xhtml">
```

Comment:
I don't think xmlns attribute is needed. You can remove it. I don't recall if I added this and for what reason.